papis/commands/add.py

```diff
@@ -306,7 +306,7 @@ def run(paths: List[str],
         temp_path = os.path.join(out_folder_path, folder_name)
         components = []     # type: List[str]
         while (
-                temp_path != out_folder_path
+                not papis.utils.paths_are_identical(temp_path, out_folder_path)
```

Comment:
Maybe you can stick a [`os.path.normpath`](https://docs.python.org/3/library/os.path.html#os.path.normpath) in front of `temp_path` and `out_folder_path` before starting this loop instead of using `paths_are_identical`? That would avoid them getting cast to a `Path` over and over again and handle the "trailing backslash" issue.

Reply:
This should work, too and is easier as you pointed out. Thank you very much.

Reply:
Perfect, thank you!